packages/database/src/migrations/20210503020225-LESMISReportsNewPermissionGroup-modifies-data.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
```

Comment:
I'm not 100% confident on this because I'm not sure what transpiler the migrations go through, but try
```suggestion
  return record.rows[0]?.id;
```